Prace_domowe/Praca_domowa1/Grupa1/Sebastian_Deregowski/PD1_Deregowski_Sebastian.ipynb

```diff
@@ -0,0 +1,2791 @@
+{
```

Comment:
<p>Doceniam, że zaadresowano problem outlierów - zwłaszcza, że nie było to wymagane w ten pracy domowej</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/39/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>